src/oncall/scheduler/default.py

```diff
@@ -486,6 +426,13 @@ def populate(self, schedule, start_time, dbinfo):
         handoff = start_epoch + timedelta(seconds=first_event_start)
         handoff = timezone(schedule['timezone']).localize(handoff)
 
+        # create temporary table if function is being used to preview
+        if req.method == 'GET':
+            start__lt = req.get_param('start__lt', required=True)
+            end__ge = req.get_param('end__ge', required=True)
+            query = 'CREATE TEMPORARY TABLE IF NOT EXISTS `%s` AS (SELECT * FROM `event` WHERE `start` < %s AND `end` > %s)' % (table_name, start__lt, end__ge)
```

Comment:
we want to interpolate these values using arguments cursor.execute to prevent sql injection problems

Reply:
Hard coded the table name and changed the interpolation to use the cursor.execute function.